regression/run-single.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+#Defines PARALLEL, getDir, assertExe, assertUsage and xargsRetCode2Msg
+. ../lib.sh
+
+assertUsage "compare" "$@"
+
+# Overrides the default pass level for all tests
+export ART_REG_TOLERANCE="${ART_REG_TOLERANCE:-2}"
+
+cmpExe="$1"
+assertExe "$cmpExe"
+testDir="$(getDir "$2")"
+scale="$(cat "$testDir"scale 2>/dev/null || echo 1)"
```

Comment:
imo this would be clearer as `cat "${testDir}scale"` or `cat "$testDir/scale"`

Reply:
done